### PR TITLE
Switch back to Hoek.clone from lodash.deepClone.

### PR DESCRIPTION
**Pull request by Invader444**
Fix lint-md script in package.json to work on Windows.

### CODE REVIEW

**Comment by geek** [comment]
@Invader444 Awesome! Yes please update this PR with 
```
os:
  - windows
  - linux
  - osx
```

for travis

**Comment by Invader444** [comment]
Well.. that is bizarre! I have no such errors in my test output! Ill look into this more tomorrow :/

**Comment by Marsup** [comment]
Classic `\n` -> `\r\n` on windows :man_facepalming:

**Comment by Invader444** [comment]
Ohhh. Git needs to checkout without auto crlf?

**Comment by Marsup** [comment]
We rather need to test with regexps matching both cases.

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,9 @@ node_js:
   - "11"
   - "node"
 
+os:
+  - linux
+  - osx
+  - windows
```
Comment (hueniverse) [inline .travis.yml]: Travis can test windows?

Comment (geek) [reply .travis.yml]: Ya, they just recently added support: https://blog.travis-ci.com/2018-10-11-windows-early-release

**Comment by Invader444** [comment]
lol!
```
lib/reporters/console.js missing coverage on line(s): 409-411, 413, 416-418, 420
Code coverage below threshold: 99.65 < 100
```
 ::sigh::

**Comment by Invader444** [comment]
@geek, does that coverage report make sense to you? It looks like the line numbers may be inaccurate... :(

**Comment by geek** [comment]
@Invader444 that missing coverage appears to be related to the supports color tty detection. We are using an outdated version and may just need to update to the latest.

**Comment by Invader444** [comment]
With latest commit:

```
C:\Users\Jon\workspace\git\Doomtrooper\lab>npm run test
npm WARN npm npm does not support Node.js v10.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN npm npm does not support Node.js v10.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/

> lab@18.0.0 test C:\Users\Jon\workspace\git\Doomtrooper\lab
> node ./bin/_lab -fL -t 100 -m 3000



  ..................................................
  ..................................................
  ..................................................
  ..................

  .

1 tests complete
Test duration: 1 ms
No global variable leaks detected

................................
  ..................................................
  ..................................................
  ................................................

348 tests complete
Test duration: 42994 ms
No global variable leaks detected
Coverage: 100.00%
Linting results: No issues


> lab@18.0.0 posttest C:\Users\Jon\workspace\git\Doomtrooper\lab
> npm run lint-md

npm WARN npm npm does not support Node.js v10.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN npm npm does not support Node.js v10.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/

> lab@18.0.0 lint-md C:\Users\Jon\workspace\git\Doomtrooper\lab
> eslint --config hapi --rule "strict: 0, eol-last: 0" --plugin markdown --ext md --parser-options "ecmaVersion: 9" .


C:\Users\Jon\workspace\git\Doomtrooper\lab>
```

**Comment by Invader444** [comment]
```
lib/reporters/console.js missing coverage on line(s): 409-411, 413, 416-418, 420
Code coverage below threshold: 99.65 < 100
```

@geek any other ideas? :(

**Comment by geek** [comment]
@Invader444 thoughts on removing windows from travis until we can sort out the build issue? If we revert the travis change then I'll go ahead and merge/release and we can come back to build issues.

**Comment by geek** [comment]
@Invader444 thanks a lot for your help. This is now published as 18.0.1

**Pull request merged**